lib/addons/flagger/my_new_addon.ts

```diff
@@ -0,0 +1,65 @@
+// filename=lib/kubevious_addon.ts
+import 'source-map-support/register';
```

Comment:
please use consistent naming for files. this is not a hello world project, it is real production level code. should be `index.ts`.

Reply:
I assume this refers to the name my_new_addon.ts that should be index.ts, my bad.